### ISSUE TITLE
[Feature request] Get description from video and put it into mkv

### ISSUE THREAD

**Issue by superbonaci** [ihabunek/twitch-dl#129]
This video
https://www.twitch.tv/videos/1260955824

has the following description, which was not embedded into mkv by twitch-dl:
```
Reaching the 20k followers goal!
```

**Comment by superbonaci** [comment]
Some videos can have description in multiple lines:

- https://github.com/lay295/TwitchDownloader/issues/830
- https://github.com/lay295/TwitchDownloader/issues/835

**Comment by ihabunek** [comment]
Implemented since 2.7.1.

e.g. here's one with a multi-line description:

```
λ twitch-dl download https://www.twitch.tv/r0dn3y/video/2223719525 -q source --end 00:05
```

```
λ ffprobe 2024-08-14_2223719525_rodney_qsv_av1_1440p_minecraft_sable.mkv

  Metadata:
    title           : [QSV AV1 1440p] Minecraft + Sable
    ENCODED_BY      : twitch-dl
    DATE            : 2024-08-14
    ARTIST          : Rodney
    DESCRIPTION     : *   1440p60 AV1 @ 12 Mbps (QSV)
                    : 
                    : *   1080p60 HEVC @ 8 Mbps (QSV)
                    : 
                    : *   720p60 H.264 @ 3.5 Mbps (QSV)
                    : 
                    : *   480p30 H.264 @ 1 Mbps (QSV)
    SHOW            : Minecraft
    ENCODER         : Lavf60.16.100
...



```
